tests/unit/client.py

```diff
@@ -296,9 +297,16 @@ def test_list_nodes_all(self):
                 ]
 
     def test_node_register(self):
-        """(successful) to node_register"""
+        """(successful) to call node_register"""
```

Comment:
(successful) **call to** node_register

not the other way around. But since you are testing unsuccessful calls in this test as well, you could just update the docstring to "Test node_register"  or "Test various calls to node_register", whatever you like.

Reply:
cannot think :( . Thanks!